tests/conftest.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+
+# See https://stackoverflow.com/a/61193490/217945 for run_slow
+def pytest_addoption(parser):
+    parser.addoption(
+        "--runslow", action="store_true", default=False, help="run slow tests"
+    )
```

Comment:
FYI Transformers has a `slow` decorator that runs tests only if the `RUN_SLOW` env variable is equal to 1 or `true`: https://github.com/huggingface/transformers/blob/6cdbd73e01a9719bfaec07d91fd108e8d932bbbb/src/transformers/testing_utils.py#L293
Not sure if that's exactly what you want to do here but bringing it up in case it saves a few lines of code.

Reply:
Good to know. Their decorator seems to be used together with `unittest`, I am using `pytest` here, so I think it does not apply.

Reply:
It should work with `pytest`, I've never got any issue in Optimum Habana with this. But up to you.

Reply:
As discussed, it really seems tied to `unittest`. For now I will leave it the way it is.